go-apps/meep-rnis/server/logger.go

```diff
@@ -24,23 +24,148 @@
 package server
 
 import (
+	"bytes"
+	"io/ioutil"
 	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"strings"
 	"time"
 
 	log "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-logger"
+	ms "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-metric-store"
 )
 
+var nextUniqueId int32 = 1
```

Comment:
Move this HTTP logger code to a new package that implements a Handler to be injected in HTTP routes. This will allow for reuse in other microservices.

Reply:
NOTE: The logger.go file can then revert back to its unmodified version.

Reply:
done